app/helpers/decidim/vocdoni/admin/steps_helper.rb

```diff
@@ -19,6 +19,14 @@ def steps(current_step)
           end
         end
 
+        def fix_it_button_with_icon(link, icon)
+          link_to link, class: "button tiny" do
+            concat icon(icon)
+            concat " "
+            concat " #{I18n.t("decidim.vocdoni.admin.steps.create_election.errors.fix_it_text")}"
```

Comment:
I'm not familiar with the `concat` method, but maybe we could get away with a bit of string interpolation here? 
```suggestion
            "#{icon(icon)}  #{I18n.t("decidim.vocdoni.admin.steps.create_election.errors.fix_it_text")}"
```